api/src/main/java/org/openmrs/ui/framework/page/PageContext.java

```diff
@@ -189,7 +190,18 @@ public void setLocale(Locale locale) {
 
     @Override
     public void includeResource(Resource resource) {
-        resourcesToInclude.add(resource);
+    	Iterator<Resource> iterator = resourcesToInclude.iterator();
+    	while (iterator.hasNext()) {	
+    		Resource res = iterator.next();

```

Comment:
Can you fix the indention in this method?

Reply:
I wonder what's wrong with git?

This is what I have locally.


![screen shot 2018-08-20 at 2 44 40 pm](https://user-images.githubusercontent.com/26084581/44338821-dd96fc00-a487-11e8-8518-d5c2c1fc4534.png)

Reply:
That's may be some kind of issues in your IDE. Better to check with another IDE :smile: 
BTW, see [here](https://github.com/openmrs/openmrs-module-uiframework/blob/b7a9afc540712f68d995520a72e36c87e64046cd/api/src/main/java/org/openmrs/ui/framework/page/PageContext.java#L195)